internal/service/project/model_project_test.go

```diff
@@ -540,3 +545,43 @@ func TestUpdateProjectBool(t *testing.T) {
 		})
 	}
 }
+
+func TestNewAdminTags(t *testing.T) {
```

Comment:
this is correct and what we're doing. However I'm seeing a slightly different approach that I like more with name being the key for a map, e.g.:

	testCases := map[string]struct {
		plan     types.Map
		expected []admin.ResourceTag
	}{
		"tags null": {types.MapNull(types.StringType), []admin.ResourceTag{}},
...

Reply:
I like your suggestion 😄 fixed in 35d729e4e186219cab55b274cb1af334b82e1645